.github/workflows/ci-test-systemd.yml

```diff
@@ -18,46 +18,59 @@ on:
 
 jobs:
   build:
-    name: Systemd Test
+    name: Test KubeArmor in Systemd Mode
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, ubuntu-18.04]
     steps:
-      - name: Kernel version
-        run: uname -a
-
       - uses: actions/checkout@v2
         with:
           submodules: true
 
-      - name: Set up Go
-        uses: actions/setup-go@v2
+      - uses: actions/setup-go@v3
         with:
-          go-version: 1.17
+          go-version: v1.18
+
+      - name: Install the latest LLVM toolchain
+        run: |
+          . /etc/os-release
+
+          sudo apt-get -y install build-essential libelf-dev pkg-config
+          wget https://apt.llvm.org/llvm.sh -O /tmp/llvm.sh
+
+          if [ "$VERSION_CODENAME" == "focal" ] || [ "$VERSION_CODENAME" == "bionic" ]; then
+            sudo bash /tmp/llvm.sh 12
+            for tool in "clang" "llc" "llvm-strip"; do
+              sudo rm -f /usr/bin/$tool
+              sudo ln -s /usr/bin/$tool-12 /usr/bin/$tool
+            done
+          else # VERSION_CODENAME == jammy
+            sudo bash /tmp/llvm.sh 14
+            for tool in "clang" "llc" "llvm-strip"; do
+              sudo rm -f /usr/bin/$tool
+              sudo ln -s /usr/bin/$tool-14 /usr/bin/$tool
+            done
+          fi
+
+      - name: Compile libbpf
+        run: |
+          cd KubeArmor/BPF
+          make -C libbpf/src
```

Comment:
Hey @nam-jaehyun should we keep this in script and invoke the script from all the actions? The script can be kept in `.github/scripts` folder.